packages/experiments/src/components/Stack/docs/StackOverview.md

```diff
@@ -0,0 +1,17 @@
+A Stack is a container-type component that abstracts the implementation of a flexbox in order to define the layout of its children components.
+
+## Stack Properties
+
+Although Stack has a number of different properties, there are three in particular that define the overall layout that the component has:
+
+- Direction: Refers to whether the stacking of children components is horizontal or vertical. By default the Stack component is vertical, but can be turned horizontal by adding the `horizontal` property when using the component.
+- Alignment: Refers to how are the children components aligned inside the container. This is controlled via the `verticalAlign` and `horizontalAlign` properties.
+- Spacing: Refers to the space that exists between children components inside the Stack. This is controlled via the `gap` ad `verticalGap` properties.
+
```

Comment:
Spelling error here, `ad` => `and`

Reply:
Don't know how that got past me, I read the text about 3 times before committing. Thanks!